src/index.jst

```diff
@@ -53,11 +54,10 @@ module.exports = function equal(a, b) {
       a instanceof Uint32Array ||
       a instanceof Float32Array ||
       a instanceof Float64Array ||
-      a instanceof BigInt64Array ||
-      a instanceof BigUint64Array
+      (envHasBigInt64Array && (a instanceof BigInt64Array || a instanceof BigUint64Array))
```

Comment:
This one fixes the #35 
All other typed array been introduced simultaneously, so their usage in ES6 context is safe, unlike `BigInt64Array` and `BigUint64Array` which been added later;